tools/test.py

```diff
@@ -120,6 +119,7 @@ def main():
         wrap_fp16_model(model)
     checkpoint = load_checkpoint(model, args.checkpoint, map_location='cpu')
     if args.fuse_conv_bn:
+        from tools.fuse_conv_bn import fuse_module
```

Comment:
I'm wondering was that an import error or something that lead to this change? If so, looks like it's just hiding it rather than solving, isn't it?

Reply:
absolutely.

But what is a reason why I should make it right way now? I did not make worse. Even if it not so complicated to refactor particular case I don't know why it is needed. We don't use such feature and it is likely that we won't face with this problem anymore. Also it will be very easy to resolve merge conflicts with upstream if we don't make significant changes in these piece of code.

Reply:
I'm not asking you to do it another way, I'm asking for a clarification that I would expect to be in a PR description.

> why I should make it right way

I've heard that making something the right way is a good practice. Haven't checked that, though.